src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.cs

```diff
@@ -6,72 +6,177 @@ namespace System.Numerics.Tensors
+        /// </para>
+        /// <para>
+        /// If a value is equal to <see cref="float.NegativeInfinity"/> or <see cref="float.PositiveInfinity"/>, the result stored into the corresponding destination location is set to <see cref="float.PositiveInfinity"/>.
+        /// If a value is equal to <see cref="float.NaN"/>, the result stored into the corresponding destination location is set to <see cref="float.NaN"/>.
+        /// </para>
+        /// <para>
+        /// <paramref name="x"/> and <paramref name="destination"/> may not overlap; if they do, behavior is undefined.
+        /// </para>
+        /// <para>
+        /// This method may call into the underlying C runtime or employ instructions specific to the current architecture. Exact results may differ between different
+        /// operating systems or architectures.
+        /// </para>
+        /// </remarks>
         public static void Abs(ReadOnlySpan<float> x, Span<float> destination) =>
             InvokeSpanIntoSpan<AbsoluteOperator>(x, destination);
 
-        /// <summary>Computes the element-wise result of: <c><paramref name="x" /> + <paramref name="y" /></c>.</summary>
+        /// <summary>Computes the element-wise addition of single-precision floating-point numbers in the specified tensors.</summary>
         /// <param name="x">The first tensor, represented as a span.</param>
         /// <param name="y">The second tensor, represented as a span.</param>
         /// <param name="destination">The destination tensor, represented as a span.</param>
-        /// <exception cref="ArgumentException">Length of '<paramref name="x" />' must be same as length of '<paramref name="y" />'.</exception>
+        /// <exception cref="ArgumentException">Length of <paramref name="x" /> must be same as length of <paramref name="y" />.</exception>
         /// <exception cref="ArgumentException">Destination is too short.</exception>
-        /// <remarks>This method effectively does <c><paramref name="destination" />[i] = <paramref name="x" />[i] + <paramref name="y" />[i]</c>.</remarks>
+        /// <remarks>
+        /// <para>
+        /// This method effectively computes <c><paramref name="destination" />[i] = <paramref name="x" />[i] + <paramref name="y" />[i]</c>.
+        /// </para>
+        /// <para>
+        /// <paramref name="x"/> and <paramref name="y"/> may overlap, but neither may overlap with <paramref name="destination"/>; if they do, behavior is undefined.
+        /// </para>
+        /// <para>
+        /// If either of the element-wise input values is <see cref="float.NaN"/>, the resulting element-wise value is <see cref="float.NaN"/>.
+        /// </para>
+        /// <para>
+        /// This method may call into the underlying C runtime or employ instructions specific to the current architecture. Exact results may differ between different
```

Comment:
Add, Subtract, Multiply, and Divide are likewise cases where all platforms should be trivially deterministic with no differences in result

we should never defer to the C runtime for them

Reply:
What might vary is the exact bit pattern for some NaN, as given NaN + NaN, it’s not guaranteed which NaN you get back